docs/user-guide/cli-configuringcli-ev.md

```diff
@@ -4,13 +4,17 @@ This section explains how to configure Zowe CLI using environment variables.
 
 By default, Zowe CLI configuration is stored on your computer in the `C:\Users\user01\.zowe` directory. The directory includes log files, profile information, and installed CLI plug-ins. When troubleshooting, refer to the logs in the `imperative` and `zowe` folders.
 
-**Note:** For information on how to define Zowe CLI environment variables to execute commands more efficiently, see [Using environment variables](cli-using-using-environment-variables.md).
+:::note
+
+For information on how to define Zowe CLI environment variables to execute commands more efficiently, see [Using environment variables](cli-using-using-environment-variables.md).
```

Comment:
It might be clearer to change this to something like:
"For information on how to use environment variables to execute commands more efficiently, see ..."

This page (cli-configuringcli-ev.md) talks about pre-defined Zowe CLI-specific environment variables that affect how Zowe CLI works. The other page (cli-using-using...) talks about how the user can define their own environment variables to make Zowe CLI more convenient or facilitate use within an automation server/pipeline.